vllm/model_executor/layers/fused_moe/configs/E=8,N=3584,device_name=NVIDIA_H100_80GB_HBM3,dtype=float8.json

```diff
@@ -127,14 +95,6 @@
         "num_warps": 8,
         "num_stages": 4
     },
-    "3072": {
```

Comment:
Is 1536 and 3072 removed on purpose?

Reply:
Good catch! I mistakenly omitted them in the previous PR. Added them back in #5242.